File: Programmers/byeonguk/kthNumber.java
import java.util.Arrays;
import java.util.ArrayList;

class kthNumber {
    public int[] solution(int[] array, int[][] commands) {
        ArrayList<Integer> answer = new ArrayList<>();
        for (int i = 0; i < commands.length; i++) {
            int[] arr = Arrays.copyOfRange(array, commands[i][0] - 1, commands[i][1]);
            Arrays.sort(arr);
            answer.add(arr[commands[i][2] - 1]);
        }
        return answer.stream().mapToInt(i->i).toArray();
    }
}

File: Programmers/byeonguk/kthNumber.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)):
        start = commands[i][0]
        end = commands[i][1]
        k = commands[i][2]

        new_array = array[start-1:end]
        new_array.sort()
        answer.append(new_array[k-1])

    return answer